kie-spring/src/test/java/org/kie/spring/CodeVersion.java

```diff
@@ -18,6 +18,6 @@
 public class CodeVersion {
 
     // Do not modify this: this is automatically incremented by the maven replacer plugin
-    public final static String VERSION = "7.0.0-SNAPSHOT";
+    public final static String VERSION = "7.1.0-SNAPSHOT";
```

Comment:
This came in when I did a rebase from git@github.com on my feature branch.  @mswiderski Should I squash this from my commit?